smarts/env/multi_scenario_v0_env.py

```diff
@@ -0,0 +1,307 @@
+# Copyright (C) 2022. Huawei Technologies Co., Ltd. All rights reserved.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+import os
+import pathlib
+from typing import Any, Dict, Optional, Tuple
+
+import gym
+
+from smarts import sstudio
+from smarts.core.agent_interface import (
+    OGM,
+    RGB,
+    AgentInterface,
+    DoneCriteria,
+    DrivableAreaGridMap,
+    NeighborhoodVehicles,
+    RoadWaypoints,
+    Waypoints,
+)
+from smarts.core.controllers import ActionSpaceType
+from smarts.env.hiway_env import HiWayEnv
+from smarts.zoo.agent_spec import AgentSpec
+
+
+def multi_scenario_v0_env(
```

Comment:
I think that @sah-huawei mentioned this earlier to me and I also agree. The file name should not contain version number even though this method can.

Reply:
Agreed. 

Changed: 
+ `smarts/env/multi_scenario_v0_env.py` to `smarts/env/multi_scenario_env.py`
+ `smarts/env/intersection_v0_env.py` to `smarts/env/intersection_env.py`